src/CTA.WebForms2Blazor/Helpers/ControlHelpers/ViewLayerControlAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace CTA.WebForms2Blazor.Helpers.ControlHelpers
+{
+    public class ViewLayerControlAttribute
+    {
+        public string Name { get; }
+        public string Value { get; }
+
+        public ViewLayerControlAttribute(string name, string value)
+        {
+            Name = name;
+            Value = value;
+        }
+
+        public override string ToString()
+        {
+            return Name + "=" + Value;
+        }
+
+        private bool Equals(ViewLayerControlAttribute other)
+        {
+            if (other is null)
+            {
+                return false;
+            }
+            return this.Name == other.Name;
```

Comment:
For string comparison please use .EuqalsIgnoreCase or the non ignore case one as needed.